docs/tutorials/intro_tutorial.ipynb

```diff
@@ -50,20 +81,22 @@
     "To install Mesa, simply:\n",
     "\n",
     "```bash\n",
-    "    $ pip install mesa\n",
+    "    $ pip install --upgrade mesa\n",
     "```\n",
     "\n",
     "When you do that, it will install Mesa itself, as well as any dependencies that aren't in your setup yet. Additional dependencies required by this tutorial can be found in the **examples/boltzmann_wealth_model/requirements.txt** file, which can be installed directly form the github repository by running:\n",
     "\n",
     "```bash\n",
-    "    $ pip install -r https://raw.githubusercontent.com/projectmesa/mesa/main/examples/boltzmann_wealth_model/requirements.txt\n",
+    "    $ pip install --upgrade -r https://raw.githubusercontent.com/projectmesa/mesa/main/examples/boltzmann_wealth_model/requirements.txt\n",
     "```\n",
```

Comment:
@EwoutH  this link is broken since we moved the examples to the mesa-examples repo. 

Could you fix the [boltzman wealth](https://github.com/projectmesa/mesa-examples/blob/main/examples/boltzmann_wealth_model_network/requirements.txt) link and instead of adding in seaborn to the Mesa dependencies why not just add seaborn to the  requirements.txt file. (Although we need to update this a pipfile) but that can be later on down the road and should not hold up this merge.

Reply:
Good catch, done!